NumbersSubtraction.java

```diff
@@ -0,0 +1,6 @@
+public class NumbersSubtraction {
+    public static void main(string args[]){

```

Comment:
@adityabyreddy `String` is mis-spelled. It should be `String`. Please check.

Reply:
@anushat846 Good catch. Fixed it.